net-im/zoom/zoom-5.5.7011.0206-r2.ebuild

```
@@ -91,7 +91,13 @@ src_install() {
 	doins *.pcm *.sh Embedded.properties version.txt
 	doexe zoom zopen ZoomLauncher
 	dosym8 -r {"/usr/$(get_libdir)",/opt/zoom}/libmpg123.so
-	dosym8 -r {"/usr/$(get_libdir)",/opt/zoom}/libquazip.so
+
+	local quazip_so="libquazip1-qt5.so"
+	if has_version "<dev-libs/quazip-1.0"; then
+		quazip_so="libquazip5.so"
+	fi
+	dosym8 -r "/usr/$(get_libdir)/${quazip_so}" /opt/zoom/libquazip.so
+
 	dosym8 -r "/usr/$(get_libdir)/libfdk-aac.so.2" /opt/zoom/libfdkaac2.so
```

Minor nitpick: Keep the unconditional links together, then the conditional ones.
```suggestion
	dosym8 -r "/usr/$(get_libdir)/libfdk-aac.so.2" /opt/zoom/libfdkaac2.so

	local quazip_so="libquazip1-qt5.so"
	if has_version "<dev-libs/quazip-1.0"; then
		quazip_so="libquazip5.so"
	fi
	dosym8 -r "/usr/$(get_libdir)/${quazip_so}" /opt/zoom/libquazip.so
```

The dependency should be updated to `dev-libs/quazip:0=` I guess.

Also, the change needs to be applied to zoom-5.4.53391.1108-r1.ebuild because that's the latest available version for x86.

---

thanks for the feedback.

---

Now you've applied the changes to 5.5.6981.0202 which isn't strictly necessary (removal candidate after next release).
We need them for zoom-5.4.53391.1108-r1.ebuild which is the latest (and only) x86 version.